explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -82,7 +86,34 @@ function StatusCard({ signature }: Props) {
     }
   }, [signature, clusterStatus]); // eslint-disable-line react-hooks/exhaustive-deps
 
-  if (!status || status.status === FetchStatus.Fetching) {
+  // Manage auto-refresh
+  React.useEffect(() => {
```

Comment:
nit: This effect is now solely determining the value of `autoRefreshInProcess` from `status`. In doing so, it sets the state of the component which causes a re-render. Instead, you could keep `autoRefreshInProcess` as a local computed variable which would remove the re-render. If you wanted to optimize the computation of the value of `autoRefreshInProcess`, you could switch to using the `React.useMemo` hook. But in this case, the computation is pretty simple so not needed.

Reply:
If you end up pulling `autoRefreshInProcess` state up into `TransactionDetailsPage` feel free to ignore this comment btw